### DISCUSSION TITLE
ca not  build dbeaver in eclipse

### DISCUSSION THREAD

**Discussion by ccczhouxin** [dbeaver/dbeaver#19922] [DBeaver Development]
### Description

![image](https://github.com/dbeaver/dbeaver/assets/43168824/e958a200-841b-4716-bd83-adc7124a7cc1)
I update maven still has error

### DBeaver Version

23.0.3

### Operating System

windows10

**Comment by Waldenth** [comment]
me too, I doubt they haven't even tried compiling according to the documents they wrote
